src/poliastro/core/elements.py

```diff
@@ -8,8 +13,63 @@
 
 @jit
 def rv_pqw(k, p, ecc, nu):
-    """Returns r and v vectors in perifocal frame.
+    r"""Returns r and v vectors in perifocal frame.
+
+    .. math::
+
+        \vec{r} = \frac{h^2}{\mu}\frac{1}{1 + e\cos(\theta)}\begin{bmatrix}
+        \cos(\theta)\vec{i}\\
```

Comment:
I don't think these `i`, `j` are needed if you are already using a column in parentheses, don't you think? (It applies to a couple of functions more)

Reply:
Oh, you are completely right @Juanlu001, this should not  appear in the column vector, since each row corresponds to each component. I will correct it immediately, not only for `\vec{r}` but also for `\vec{v}`